electron/preload.js

```diff
@@ -1,2 +1,4 @@
-const { ipcRenderer } = require('electron')
-window.ipcRenderer = ipcRenderer
\ No newline at end of file
+const { ipcRenderer, clipboard, nativeImage } = require('electron')
```

Comment:
Since there is also use of node in `imageGrid.vue`- should we refactor that and import that stuff there or keep as is? Think now that `preload.js` is being used it may make sense? Better than having them at two places right?

Reply:
You are right, it's my first time using electron and it threw an error so I thought it had to be done this way.